ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -340,6 +340,18 @@ g_template_openshift_master:
     url: 'https://github.com/openshift/ops-sop/blob/master/V3/Alerts/openshift_master.asciidoc'
     priority: info
 
+  - name: 'Failed Persistent Volumes on {HOST.NAME}'
+    expression: '{Template Openshift Master:openshift.master.pv.failed.count.last(#2)}>0'
```

Comment:
I would do `.min(#2) > 0` instead of .last. I try to not use .last as .min will helps with flapping better.

Unless there's a reason .last is better in this specific case.